Feature/course 1

grojas123 opened the pull request:
"Fixed the nctss_3.css for task2" Final PR of Tasks 2 .

manuelbarzi commented on staff/luca-lamorte/feature/course-1/NYSL/home.html:
```diff
@@ -0,0 +1,51 @@
+
+<body>
+    <header class="header">
+        <div class="hero">
```
is this wrapping div (hero) necessary?

manuelbarzi commented on staff/luca-lamorte/feature/course-1/NYSL/home.html:
```diff
@@ -0,0 +1,51 @@
+            </ul>
+        </nav>
+
+        <div class="events">
```
may this be a section?

manuelbarzi commented on staff/luca-lamorte/feature/course-1/NYSL/home.html:
```diff
@@ -0,0 +1,51 @@
+        </nav>
+
+        <div class="events">
+            <div class="event-container">
```
may you list events in a ul element?

RoseKoning commented on staff/luca-lamorte/feature/course-1/NYSL/about-us.html:
```diff
@@ -0,0 +1,49 @@
+
+<body>
+    <header class="header">
+        <div class="hero">
```
You might not need a div for a <h1> element in a header

RoseKoning commented on staff/luca-lamorte/feature/course-1/NYSL/about-us.html:
```diff
@@ -0,0 +1,49 @@
+            </ul>
+        </nav>
+
+        <div class="about-us-container">
```
this could also be a < section> element

manuelbarzi commented:
There has been a change in the `upstream` repo, please, re-do this PR against your new branch `develop-<your-program-year-month`>. Thank you!

grojas123 committed:
Added the AssetsOf The website

grojas123 committed:
Removed the AssetsOf The website

grojas123 committed:
Removed the AssetsOf The website

grojas123 committed:
Finished the First version of the WebSite

grojas123 committed:
Improvments of the last version of the WebSite

grojas123 committed:
Improvments of the index.html

grojas123 committed:
Added the styles files

grojas123 committed:
First Version of the site with styles/nysl_1.css

grojas123 committed:
Moved First Version of the site with styles/nysl_1.css to a new folder named nysl_1.css

grojas123 committed:
Moved the files to create Second Version of the site with styles/nysl_2.css

grojas123 committed:
Moved the files to create Second Version of the site with styles/nysl_2.css

grojas123 committed:
Created the second option of the website with nysl_2.css

grojas123 committed:
Added the ids actualpage and otherpages to all HTML versions

grojas123 committed:
First version of the site with the 3 versions of .css

grojas123 committed:
First version of the site only text and some css

grojas123 committed:
First version of the site only text and some css

grojas123 committed:
Version of the site images and css

grojas123 committed:
First version with nctss_1.css

grojas123 committed:
First version with nctss_1.css complete using flexbox

grojas123 committed:
Correctec the spaces in the menu and added the vertical menus bars

grojas123 committed:
Correctec the spaces in the menu and added the vertical menus bars

grojas123 committed:
Updated the nctss_1.css with .horizontal-image-container

grojas123 committed:
First version with nctss_2.css

grojas123 committed:
Updated the \staff\gustavo-rojas\course-1-new-version\task1 with no usage of FlexBox with nctss_1.css

grojas123 committed:
Updated the \staff\gustavo-rojas\course-1-new-version\task2 with usage of FlexBox with nctss_2.css

grojas123 committed:
Updated the \staff\gustavo-rojas\course-1-new-version\task2 with usage of FlexBox with nctss_[1,2,3].css final design

grojas123 committed:
Updated the \staff\gustavo-rojas\course-1-new-version\task2 with usage of FlexBox with nctss_[1,2,3].css final design with menu fonts in white

grojas123 committed:
Final Version of \staff\gustavo-rojas\course-1-new-version\task2 with usage of FlexBox with nctss_[1,2,3].css Tested done

grojas123 committed:
Created the folder of task3

grojas123 committed:
Fixed the nctss_3.css for task2

grojas123 committed:
First version of task2 created general_info.html and all ntss_[1,2,3}.

grojas123 committed:
Task3 created the general_info.html page and updated all the menus in all the pages with rules.html too in the menu. Tested . To create rules.html

grojas123 committed:
Task3 created the general_info.html page and updated all the menus in all the pages with rules.html too in the menu. Updated the nctss_3.css

grojas123 committed:
Task3 created the general_info.html with tables. Updated the nctss_3.css

grojas123 committed:
Task3 adjusted the width of .information-container

elolivier commented on staff/gustavo-rojas/git/README.md:
```diff
@@ -0,0 +1,32 @@
+#To create a repository in the actual directory
```
Add an space after # and before first letter to be consistent in all lines. Please validate every step to follow this comment.

grojas123 replied:
Done

elolivier commented on staff/gustavo-rojas/git/README.md:
```diff
@@ -0,0 +1,32 @@
+git add .
+#To send to the commit all the files in the stagging area
+git commit -m " The comment about the change"
+# to change a branch
```
Start with capital letter to be consistent with all lines.

grojas123 replied:
Done.

elolivier commented on staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css:
```diff
@@ -0,0 +1,88 @@
+    margin: 2em
+}
+
+.nameofthevent-container {
```
Please use same convention for all class names, if you are going to separate each word with dash (-) use it everywhere. Review all class names.

grojas123 replied:
Done.

elolivier commented on staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css:
```diff
@@ -0,0 +1,88 @@
+    visibility: collapse;
+}
+
+.actual_page .contact_title {
```
Same code as actual_page .about_title, why two classes?

grojas123 replied:
I corrected I mean .actual-page.contact-title to be more specific .

elolivier commented on staff/gustavo-rojas/course-1-new-version/task1/about.html:
```diff
@@ -0,0 +1,52 @@
+  <div class=nameofthevent-container>
+    <p class="nameoftheevent"> NorCal Time Trial Series</p>
+  </div>
+  <div class="horizontal-image-container">
```
What is the meaning of this div, it is iin all pages but i cannot see the image.

grojas123 replied:
Removed the division .

grojas123 committed:
Task1 adjusted the comments of PR. Corrected the README.mdRemoved the <div class=horizontal-image-container> and changed the way how the classes are named

grojas123 commented:
Task1 adjusted the comments of PR. Corrected the README.md . Removed the <div class=horizontal-image-container> and changed the way how the classes are named

grojas123 commented:
Need to fix one error.
